### ISSUE TITLE
TP54001819 on T22_V06 20180705 burning

### ISSUE THREAD

**Issue by FLSCHI** [Xinyuan-LilyGO/LilyGO-T-Beam#11]
Hello,
when powering the T22_V06 20180705 with an 18650 battery which got charged in an external battery charger to exactly 4.16V will destroy the TTGO-Beam! 
The TP54001819 chip (responsible for charging?) on the TTGO-Beam board will become extremely hot until smoke rises and bye-bye-beam! (happend on 2 devices!)
Is this issue solved in later revisions or will this still be the same behaviour? 
Thanks,
FS

**Comment by lewisxhe** [comment]
V6 is an old product. When did you buy it?

**Comment by FLSCHI** [comment]
I think roundabout 1.5 years ago. 

Will this also happen with a later revision?

**Comment by lewisxhe** [comment]
There are two versions behind, one is TP5400 and the other is AXP192, which uses two different power chips

**Comment by peterwishart** [comment]
I have a similar problem with a T22_V06 which had been sitting for a while, powered off but with battery installed. Battery was down to 3.8v.
When powering via 18650 one of the components (highlighted in picture) heats up enough to smoke a little after a few seconds. Tried with a range of batteries and still get the same. 
It still works fully via USB power with no battery installed, running the last program I flashed to it.

![Pic of board showing component that heats up](https://user-images.githubusercontent.com/5476758/97296741-f611b680-1848-11eb-9074-4ae56047c596.jpg)
